docs/index.bs

```diff
@@ -3237,30 +3239,78 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
       : Input
-      :: |rule|, a {{RouterRule}}
+      :: |condition|, a {{RouterCondition}}
       :: |serviceWorker|, a [=/service worker=]
       : Output
       :: a boolean
 
       1. Let |hasCondition| be false.
-      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"] [=map/exists=], then:
-          1. Let |rawPattern| be |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"].
+      1. If |condition|["{{RouterCondition/urlPattern}}"] [=map/exists=], then:
+          1. Let |rawPattern| be |condition|["{{RouterCondition/urlPattern}}"].
           1. Let |pattern| be the result of running <a>Parse URL Pattern</a> algorithm passing |rawPattern| and |serviceWorker|. If this throws an exception, catch it and return false.
           1. If |pattern| [=URLPattern/has regexp groups=], then return false.
 
               Note: Since running a user-defined regular expression has a security concern, it is prohibited.
 
           1. Set |hasCondition| to true.
-      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/requestMethod}}"] [=map/exists=], set |hasCondition| to true.
-      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/requestMode}}"] [=map/exists=], set |hasCondition| to true.
-      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/requestDestination}}"] [=map/exists=], set |hasCondition| to true.
-      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/runningStatus}}"] [=map/exists=], set |hasCondition| to true.
+      1. If |condition|["{{RouterCondition/requestMethod}}"] [=map/exists=], set |hasCondition| to true.
+      1. If |condition|["{{RouterCondition/requestMode}}"] [=map/exists=], set |hasCondition| to true.
+      1. If |condition|["{{RouterCondition/requestDestination}}"] [=map/exists=], set |hasCondition| to true.
+      1. If |condition|["{{RouterCondition/runningStatus}}"] [=map/exists=], set |hasCondition| to true.
+      1. If |condition|["{{RouterCondition/_or}}"] [=map/exists=], then:
+          1. If |hasCondition| is true, return false.
+
+              Note: For ease of understanding the router rule, the "or" condition is mutual exclusive with other conditions.
+
+          1. Let |or conditions| be |condition|["{{RouterCondition/_or}}"].
+          1. For each |or condition| of |or conditions|:
+              1. If running [=Verify Router Condition=] algorithm with |or condition| and |serviceWorker| returns false, return false.
+          1. Set |hasCondition| to true.
       1. Return |hasCondition|.
   </section>
 
+  <section algorithm>
+    <h3 id="match-router-condition-algorithm"><dfn>Match Router Condition</dfn></h3>
+      : Input
+      :: |condition|, a {{RouterCondition}}
+      :: |serviceWorker|, a [=/service worker=]
+      :: |request|, a [=/request=]
+      : Output
+      :: a boolean
+
+      Note: if there are multiple conditions, all conditions will be matched to return the source.
```

Comment:
This note probably makes more sense inside the loop in Get Router Source, instead of here in the algorithm that's focused on only a single condition?

Reply:
This explanation has been added as an answer to https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/5#discussion_r1453049376.

People can write something like:
```
condition: {
  urlPattern: "/test",
  runningStatus: "running"
}
```

This function will return true if both `urlPattern` and `runningStatus` are fulfilled.  I tried to explain that here.

Reply:
But return "source" sounds wrong here X(

Reply:
I have updated the explanation.